website-1.x/blog/2021-03-09-releasing-docusaurus-i18n.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Releasing Docusaurus i18n
+author: Sébastien Lorber
+authorTitle: Docusaurus maintainer
+authorURL: https://sebastienlorber.com
+authorImageURL: https://github.com/slorber.png
+authorTwitter: sebastienlorber
+tags: [release, i18n]
+image: /img/blog/2021-03-09-releasing-docusaurus-i18n/social-card.png
+---
+
+Today, we officially release **[Docusaurus 2 i18n](https://v2.docusaurus.io/docs/i18n/introduction)**, as part of [2.0.0-alpha.71](https://github.com/facebook/docusaurus/releases/tag/v2.0.0-alpha.71).
```

Comment:
Probably should explain somewhere up top that the abbreviation **i18n** stands for "internationalization", because this will not be obvious to people who are encountering this abbreviation for the first time here. 😄

Reply:
thanks, fixed